sympy/parsing/autolev/_parse_autolev_antlr.py

```diff
@@ -0,0 +1,38 @@
+import sys
+from sympy.external import import_module
+
+AutolevParser = AutolevLexer = AutolevListener = None
+try:
+    AutolevParser = import_module('sympy.parsing.autolev._antlr.autolevparser',
+                                  __import__kwargs={'fromlist': ['AutolevParser']}).AutolevParser
+    AutolevLexer = import_module('sympy.parsing.autolev._antlr.autolevlexer',
+                                 __import__kwargs={'fromlist': ['AutolevLexer']}).AutolevLexer
+    AutolevListener = import_module('sympy.parsing.autolev._antlr.autolevlistener',
+                                    __import__kwargs={'fromlist': ['AutolevListener']}).AutolevListener
+except Exception:
```

Comment:
This should be a specific exception.

Reply:
I copied this style from the LaTeX parser.
I think ImportError should work though.